dockerunit-consul/src/main/java/com/github/qzagarese/dockerunit/discovery/consul/ConsulDiscoveryProvider.java

```diff
@@ -68,10 +66,10 @@ public ServiceContext clearRegistry(ServiceContext currentContext, ServiceContex
 	}
 
 	private Service doDiscovery(Service s) {
-		List<SrvRecord> records;
+		List<ServiceRecord> records;
 		try {
-			records = resolver.resolveSRV(s.getName() + CONSUL_DNS_SUFFIX, s.getInstances().size(), 
-					discoveryTimeout, dnsPollingFrequency);
+			records = resolver.resolveService(s.getName(), s.getInstances().size(), 
+					discoveryTimeout, consulPollingPeriod);
 		} catch (Throwable t) {
```

Comment:
Usually this is a big no no maybe a @supresswarnings("squid:s1181") wouldn't be a bad idea. https://sbforge.org/sonar/rules/show/squid:S1181?layout=false

Reply:
:+1: